app/Notifications/RequestWithdrawalApprovedNotification.php

```
@@ -0,0 +1,73 @@
+    public function __construct($token, RequestWithdrawal $model)
+    {
+        $this->token = $token;
+        $this->model = $model;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @return array<int, string>
+     */
+    public function via(object $notifiable): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (undefined)

(UnusedFormalParameter)